api/HTMLInputElement.json

```
@@ -210,7 +210,8 @@
               "version_added": null
             },
             "ie": {
-              "version_added": "10"
+              "version_added": "10",
+              "notes": "Read-only; setting of this property not supported."
```

Thanks for addressing this, @FND and welcome to the BCD project!

We can probably simplify this a little by flagging the partial implementation, like this:

```suggestion
              "partial_implementation": true,
              "notes": "This property is read-only."
```

---

Thanks for improving this; I've applied your fix retroactively in 81f4bddfd46d7c32404c68d99627dcfc9082f178.

(FWIW, I'd grepped for "read-only" and copied the phrasing from [`api/CSSStyleRule.json`](https://github.com/mdn/browser-compat-data/blob/master/api/CSSStyleRule.json#L73) for consistency. 🤷‍♂️ )

---

oh yeah, inconsistency in notes is a thing I'm trying to improve. Another one for my todo list 👍

---

Excellent, I appreciate that.